dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/ConsoleHelper.scala

```diff
@@ -53,7 +53,9 @@ class ConsoleHelper(colors: Colors) {
 
       println()
       // and finally shut down any other possibly running embedded server
-      Await.result(Servers.asyncTryStop(log), 60.seconds)
+      Await.result(Future.traverse(infrastructureServices)(serv => Future {
+        serv.close()
+      }), 60.seconds)
```

Comment:
Previously, Kafka, Cassandra and the service locator were shutdown based on their instances in global state. Now, their start tasks return a `Closeable` to shut them down, which is passed in here. This is so that other custom start tasks can be defined, so they can all be shutdown together rather than the hard coded shutdown sequence that used to be used.